### ISSUE TITLE
how to get the loss after each iteration

### ISSUE THREAD

**Issue by Ddogzhouyuan** [Azure/mmlspark#951]
Hello, I want to get the loss trace of lightgbm model to plot, but I don't find any API to get the loss of each iteration. Can you help me or give me any suggestion?. Thanks.

**Comment by ranadeepsingh** [comment]
Thank you for reaching out. Per-iteration training loss metrics can be retrieved by setting `setIsProvideTrainingMetric(true)` (or `isProvideTrainingMetric=True` in Python) on your LightGBM estimator. As this support question has been answered, we are closing this issue.
